mesheryctl/internal/cli/root/mesh/deploy.go

```diff
@@ -2,140 +2,56 @@ package mesh
 
 import (
 	"fmt"
-	"io"
 	"net/http"
-	"net/url"
 	"strings"
 	"time"
 
 	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
 	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
-	smp "github.com/layer5io/service-mesh-performance/spec"
-	"github.com/manifoldco/promptui"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
 )
 
-// checkArgs checks whether the user has supplied an adapter(-a) argument
-func checkArgs(n int) cobra.PositionalArgs {
-	return func(cmd *cobra.Command, args []string) error {
-		if len(args) < n || args[0] == "" {
-			return fmt.Errorf("the '-a' (Adapter to use for installation) argument is required in the mesh command")
-		}
-		return nil
-	}
-}
-
-var (
-	meshName  string
-	deployCmd = &cobra.Command{
-		Use:   "deploy",
-		Short: "Deploy a service mesh to the Kubernetes cluster",
-		Args:  checkArgs(1),
-		Long:  `Deploy a service mesh to the connected Kubernetes cluster`,
-		Example: `
-// Deploy a service mesh from an interactive on the default namespace
+var deployCmd = &cobra.Command{
+	Use:   "deploy [mesh-name]",
+	Short: "Deploy a service mesh",
+	Long:  "Deploy a service mesh in the connected Kubernetes cluster",
+	Example: `
+// Deploy a service mesh interactively:
 mesheryctl mesh deploy
 
-// Deploy Linkerd mesh on a specific namespace
-mesheryctl mesh deploy --adapter meshery-linkerd --namespace linkerd-ns
-
-// Deploy Linkerd mesh and wait for it to be deployed
-mesheryctl mesh deploy --adapter meshery-linkerd --watch
+// Deploy Linkerd in a specific namespace:
+mesheryctl mesh deploy linkerd --namespace linkerd-ns
 
-! Refer below image link for usage
-* Usage of mesheryctl mesh deploy
-# ![mesh-deploy-usage](/assets/img/mesheryctl/deploy-mesh.png)
```

Comment:
Restore might be needed

Reply:
Why? Please include more detail so I know how to respond.

Reply:
Hmm... I'm asking a curious Q 😅, is there any macro variable to store such markdown code like above, we may need them for docs so that the golang script will fetch and include the content in page during CI merge trigger

Reply:
I see. I'll add that block then when adding examples.

Perhaps something like this would work to get the content for docs: https://github.com/spf13/cobra/blob/main/doc/md_docs.md